### ISSUE TITLE
Service Worker fails to load styles.css randomly

### ISSUE THREAD

**Issue by mfabioe** [angular/angular#35610]
# 🐞 bug report

### Affected Package
I don't know

### Is this a regression?
I guess version 7 didn't have this issue. Probably related to https://github.com/angular/angular/issues/30778, we had the same problem and it seemed to have been fixed, but recently it started popping up again.

### Description
Randomly we have an error in which the css is not loaded by the ngsw-worker.js. It causes our page to be rendered without any css. It is not related to chrome's storage quota anymore, maybe it's something else.

## 🔬 Minimal Reproduction
I don't know how to reproduce, because the error appears randomly. Most of the time it works.

## 🔥 Exception or Error
<pre><code>
The FetchEvent for "url/styles.e4e81ab9916f6440b64c.css" resulted in a network error response: the promise was rejected.
ngsw-worker.js:1 Uncaught (in promise) DOMException: Entry already exists.
The FetchEvent for "url/styles.e4e81ab9916f6440b64c.css" resulted in a network error response: the promise was rejected.
ngsw-worker.js:1 Uncaught (in promise) DOMException: Entry already exists.
styles.e4e81ab9916f6440b64c.css:1 Failed to load resource: net::ERR_FAILED
</code></pre>

## 🌍  Your Environment

**Angular Version:**
<pre><code>
Angular CLI: 8.3.13
Node: 10.15.3
OS: win32 x64
Angular: 8.2.11
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router, service-worker

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.803.3
@angular-devkit/build-angular     0.803.13
@angular-devkit/build-optimizer   0.803.13
@angular-devkit/build-webpack     0.803.13
@angular-devkit/core              8.3.13
@angular-devkit/schematics        8.3.13
@angular/cdk                      8.2.3
@angular/cli                      8.3.13
@angular/material                 8.2.3
@ngtools/webpack                  8.3.13
@schematics/angular               8.3.13
@schematics/update                0.803.13
rxjs                              6.5.3
typescript                        3.5.3
webpack                           4.39.2
</code></pre>

**Anything else relevant?**
Chrome 79.0.3945.130 64 bits

**Comment by gkalpak** [comment]
Based on the error message (`DOMException: Entry already exists`) this sounds like a browser (i.e. non-Angular specific) issue. Potentially relevant Chrome bugs: [542668][1], [968444][2]

Does it happen on specific browsers/OS/version? (E.g. similar issues in the past only happened on Chrome on Windows.)

[1]: https://bugs.chromium.org/p/chromium/issues/detail?id=542668
[2]: https://bugs.chromium.org/p/chromium/issues/detail?id=968444

**Comment by fabioemoutinho** [comment]
I only have reports for this to happen on Chrome

**Comment by gkalpak** [comment]
It sounds like a Chrome issue. Does it happen on a specific OS or have you seen it in more than one?

**Comment by andreialecu** [comment]
I'm seeing this same error but with one of the core `.js` files, not a `css`. 

Angular 9.0.4
Chromium 80

This results in a white page being shown.

![image](https://user-images.githubusercontent.com/697707/76769191-ca076080-67a4-11ea-8690-93f4030e0a5b.png)

![image](https://user-images.githubusercontent.com/697707/76769205-cf64ab00-67a4-11ea-8f2a-ccaaa3a407e2.png)

I've migrated from a custom workbox based serviceworker to the angular native service worker and now running into this. Various users have reported similar issues as well.

If I press `ctrl+f5` everything loads, but the error returns randomly.

After using DevTools to debug it, I found that the error is `DOMException: Quota exceeded.`  and occurs in this block: 

![image](https://user-images.githubusercontent.com/697707/76770106-1606d500-67a6-11ea-9e1c-8352a6350fe6.png)

Any suggestions? Thanks.
